sashimi-webapp/src/components/file-manager/Documents.vue

```diff
@@ -117,29 +118,39 @@ export default {
   padding: 20px;
   box-sizing: border-box;
 
+  .folder-wrapper,
+  .file-wrapper {
+    margin: 0 30px 20px 0;
+    width: 140px;
+    height: 140px;
+    text-align: center;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
+  }
+
   .folder,
   .file   {
-    width: 120px;
-    height: 120px;
     text-align: center;
     position: relative;
-    margin-right: 30px;
+    width: 100%;
+    height: 100%;
+    background-color: white;
 
     img {
-      width: 120px;
+      width: 50px;
     }
 
     p {
+      display: block;
       font-size: $documents-name-font-size;
-      position: absolute;
-      top: 15px;
-      left: 0;
-      bottom: 0;
-      right: 0;
-      width: 60px;
-      height: 40px;
-      margin: auto;
-      overflow: auto;
+      // position: absolute;
+      // top: 15px;
+      // left: 0;
+      // bottom: 0;
+      // right: 0;
+      // width: 60px;
+      // height: 40px;
+      // margin: auto;
+      // overflow: auto;
```

Comment:
minor stuffs: I think when commenting out multiple lines it would be better to do the following instead of commenting out line by line

```
/*
position: absolute;
top: 15px;
left: 0;
bottom: 0;
right: 0;
width: 60px;
height: 40px;
margin: auto;
overflow: auto;
*/
```